add content for clinic us server (#9566)
pingcap/docs: clinic/clinic-introduction.md

    - [Clinic Server in Chinese mainland](https://clinic.pingcap.com.cn): the collected data is stored in AWS China (Beijing).
    - [Clinic Server in US](https://clinic.pingcap.com): the collected data is stored in AWS US.

## User scenarios

- Troubleshoot cluster problems remotely

    When your cluster has some problems that cannot be fixed quickly, you can ask for help at [TiDB Community slack channel](https://tidbcommunity.slack.com/archives/CH7TTLL7P) or contact PingCAP technical support. When contacting technical support for remote assistance, you need to save various diagnostic data from the cluster and forward the data to the support staff. In this case, you can use Diag to collect diagnostic data with one click. Diag helps you to collect complete diagnostic data quickly, which can avoid complex manual data collection operations. After collecting data, you can upload the data to the Clinic Server for PingCAP technical support staff to troubleshoot cluster problems. The Clinic Server provides secure storage for uploaded diagnostic data and supports the online diagnosis, which greatly improves the troubleshooting efficiency.

- Perform a quick check on the cluster status

>     Even if your cluster runs stably now, it is necessary to periodically check the cluster to avoid potential stability risks. You can check the potential health risks of a cluster using the local and remote quick check feature provided by PingCAP Clinic.

Reviewer: ```suggestion
    Even if your cluster runs stably now, it is necessary to periodically check the cluster to avoid potential stability risks. You can check the potential health risks of a cluster using the local and server-side quick check feature provided by PingCAP Clinic.
```

    Using SQL statements, Diag can query system variables and other information of TiDB. To use this method, you need to **additionally provide** the username and password to access TiDB when collecting data.

## The limitations of Clinic Server

> **Note:**
>
> - Clinic Server provides free service from July 15, 2022 to Jan 14, 2023. PingCAP Clinic will notify you through email before Jan 14, 2023 if the service starts requiring payment.
> - If you want to adjust the usage limitations, contact PingCAP technical support.

| Service Type| Limitation |
| :------ | :------ |
| Number of clusters | 10/organization |
| Storage capacity | 50 GB/cluster |
| Storage time | 180 days |

> | Data size | 3 GB |

Reviewer: ```suggestion
| Data size limit per package | 3 GB |
```

> | Saving time | 3 days|

Reviewer: ```suggestion
| Data rebuild environment saving time | 3 days|
```
